### ISSUE TITLE
About the results

### ISSUE THREAD

**Issue by EvanSong77** [wireless911/span-aste#2]
For this dataset 15res, the bilstm version(your code) i got the best f1 in the eval is 0.6382,the paper is 0.6426, and the bert version i got  the best f1 in the eval is 0.6564, the paper is 0.7075. I can't get close to the results of the paper, can you give me some suggestions? Thank you very much!

**Comment by wireless911** [comment]
The result index you get in the bert version is slightly lower, my parameter selection is batchsize=4, lr=3e-5, you can try to modify your hyperparameters to see the result.

**Comment by EvanSong77** [comment]
Thank you! now, the dataset 15res can reach 0.695(best relation F1) in the eval, can you reappear the results of this paper in the dataset 15res?

**Comment by wireless911** [comment]
After the bert version I implemented was trained in the 15res dataset, the f1 value of the validation set was 69.1 and the result(70.75) in the paper was still slightly different. I am still exploring.

**Comment by codewen77** [comment]
你好, 我对你的metrics很疑惑. 为什么你用的是预测的golden label来计算而不是跟这个句子的ground truth来计算，试想一下如果你某个句子有两个三元组，而你预测出的pair都是invalid，那么按照你的计算方法，这个是肯定不参与计算的，您确定这样没问题？. 并且我统计了一下每次跑验证集的时候num_golds的个数都是远小于ground truth的个数

**Comment by Zyuting1** [comment]
hi, i tried bilstm on the 14res.txt, but i got a low score like 0.527(relation F1). i have tried to adjust the parameters but didnt work.
the parameters now are batchsize 8 and lr 3e-4.
have u tried other datasets expect 15res.txt to get a score thats close to the score in paper?
looking forward to ur reply! thank u so much!

**Comment by Resist4263** [comment]
I think there may be a problem with the F1 calculation.Because the F1 of the whole dataset is not calculated by being averaged from the F1 of each batch.

**Comment by EvanSong77** [comment]
> 我认为F1的计算可能存在问题。因为整个数据集的 F1 不是通过从每个批次的 F1 中取平均值来计算的。

yeah, you are right, he has a problem with metrics

**Comment by wireless911** [comment]
> 
正如你们提到的问题，我在最新的版本进行了修复，同时更新了bert 版本，感谢你们提出的宝贵意见
